ddtrace/contrib/elasticsearch/__init__.py

```diff
@@ -18,6 +22,23 @@
     es = Elasticsearch(port=ELASTICSEARCH_CONFIG['port'])
     Pin.override(es.transport, service='elasticsearch-videos')
     es.indices.create(index='videos', ignore=400)
+
+
+
+Configuration
+~~~~~~~~~~~~~
+
+.. py:data:: ddtrace.config.elasticsearch['service_name']
```

Comment:
```suggestion
.. py:data:: ddtrace.config.elasticsearch['service']
```

both will work - but we prefer to do service as eventually we'd like to get rid of service_name